docs/content/3.manual/manual.yml

```diff
@@ -301,6 +301,12 @@ sections:
           to the last element, -2 referring to the next to last element,
           and so on.
 
+          The `.foo` syntax only works for simply keys i.e. keys that
+          are all alphanumeric characters. `.[<string>]` works with
+          keys that contain special charactors such as colons and dots.
+          For example `.["foo::bar"]` and `.["foo.bar"]` work while
+          `.foo::bar` and `.foo.bar` would not.
```

Comment:
It's weird to say that `.foo.bar` doesn't work; it does select the key `bar` in `foo`. Perhaps qualifying that "would not", as in "would not select a key named `foo.bar`", would make it clearer?